index2.less

```diff
@@ -0,0 +1,412 @@
+// Is this working?
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+	height: 100%;
+	font-size: 62.5%;
+}
+
+//Variables
+@main-font: 'Raleway', sans-serif;
+@main-color: teal;
+
+//Mixins
+.test(){
+
+}
+.image-size(){
+	height: 100%;
+	width: auto;
+}
+
+.center() {
+	margin: 0 auto;
+}
+
+.columns(){
+	display: flex;
+	flex-direction: column;
+}
+
+.border(){
+	border-bottom: 1px dashed silver;
+}
+
+
+
+.main-container {
+	max-width: 1000px;
+	width: 100%;
+	.center;
```

Comment:
I would read the documentation regarding calling mixins: http://lesscss.org/features/#mixins-feature

"Currently and historically, the parentheses in a mixin call are optional, but optional parentheses are deprecated and will be required in a future release."

```
.a(); 
.a;  // currently works, but deprecated; don't use
```

Reply:
So when calling mixins, make sure to add the `()` at the end